adapter/redisquota/driver.go

```diff
@@ -37,7 +39,9 @@ type response struct {
 
 // Get is to get a connection from the connection pool.
 func (cp *connPool) get() (*connection, error) {
+	cp.Lock()
```

Comment:
Actually, looking at the implementation for the Redis pool functions, they already seem to be thread-safe. So it doesn't seem as though we need any logic for mutual exclusion of calls to the Redis pool functions.